src/ask-password/ask-password.c

```diff
@@ -90,6 +95,7 @@ static int parse_argv(int argc, char *argv[]) {
                 {}
         };
 
+        const char* emoji = NULL;
```

Comment:
Not a matter, and this is not strict rule, but we usually prefix `*` at variable name: `const char *emoji`.